### DISCUSSION TITLE
a/hikvision-rtp-ps-stream-parser

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#8] [Announcements]
# a/hikvision-rtp-ps-stream-parser

https://blog.yasking.org/a/hikvision-rtp-ps-stream-parser.html

**Comment by dangpengsong** [comment]
博主,你好,我在学习国标推流方面的知识,看到你的博客,里面详尽的讲解,让我受益匪浅,看到你文章里面有pes这些的结构体代码,请问这个是在哪里找到的,有这些代码对照看的话理解起来很清晰,可以指点下吗,谢谢

**Reply by sincerefly** [reply]
你好，才看到你的留言，Pes 等结构体代码是在 iso13818-1 协议中定义的，我在文章中给出的 PDF 下载地址刚看了下已经失效，本文中，我参考的版本是 2000 版。

可以通过这个地址下载：[https://ocw.unican.es/pluginfile.php/2825/course/section/2777/iso13818-1.pdf](https://ocw.unican.es/pluginfile.php/2825/course/section/2777/iso13818-1.pdf)
